lib/saluki-io/src/net/client/http/client.rs

```diff
@@ -105,6 +105,7 @@ where
 /// - support for FIPS-compliant cryptography (if the `fips` feature is enabled in the `saluki-tls` crate) via [AWS-LC][aws-lc]
 ///
 /// [aws-lc]: https://github.com/aws/aws-lc-rs
+#[derive(Clone)]
```

Comment:
Does this still need to be `Clone`?

(Ditto for `HttpsCapableConnectorBuilder` and `ClientTLSConfigBuilder`.)

Reply:
removed 👍